srcpkgs/snes9x-gtk/template

```diff
@@ -0,0 +1,20 @@
+# Template file for 'snes9x-gtk'
+pkgname=snes9x-gtk
+version=1.56.2
+revision=1
+wrksrc=snes9x-$version/gtk/
+build_style=gnu-configure
+hostmakedepends="automake autoconf pkg-config"
+makedepends="intltool SDL2-devel minizip-devel gtk+3-devel gettext gettext-devel glib-devel alsa-lib xorg-server-devel libxml2-devel"
```

Comment:
This can be reduced to 

~~~
makedepends="gettext-devel glib-devel gtk+3-devel intltool minizip-devel SDL2-devel
 xorg-server-devel"